packages/picker/src/Picker.ts

```diff
@@ -96,6 +96,14 @@ export class PickerBase extends SizedMixin(Focusable, { noDefaultSize: true }) {
     @property({ type: Boolean, reflect: true })
     public invalid = false;
 
+    /**
+     * Forces the Picker to render as a popover on mobile instead of a tray.
+     *
+     * @memberof PickerBase
```

Comment:
Using `@memberof` is a nice touch! While our current documentation strategy doesn’t currently pick this up, it’s something we could consider leveraging in the future.